test/auth.spec.js

```diff
@@ -12,10 +12,11 @@ describe("client authentication", function () {
         });
     });
 
-    helper.allTests(function(parser, ip, args) {
+    helper.allTests({
+        allConnections: true
+    }, function(parser, ip, args) {
```

Comment:
The test suite has a new default to only run each test with each parser once instead of each parser * each connection type (IPv4, IPv6 and socket). This reduces unnecessary overhead. To run each connection type this optional option may be passed to the helper.